hxmath/math/Quaternion.hx

```diff
@@ -32,6 +32,38 @@ class QuaternionDefault
     }
 }
 
+// yaw (Z), pitch (Y), roll (X)
+@:forward
+abstract Euler( Vector3 ) from Vector3 to Vector3 {
+    public inline function new( roll: Float, pitch: Float, yaw: Float ){
```

Comment:
Minor: order fields like existing structures (properties on top, make setter/getter implementations private and put them on the bottom).